.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@
 ## QA
 
 - Pull code
-- Run `./run clean && ./run serve`
+- Run `dotrun clean && dotrun serve`
```

Comment:
fwiw I would always just run this locally with yarn as there are no global dependencies. `yarn && yarn start`

Reply:
Ohh that's a good point. Thanks @hatched 🎉